Libraries/StyleSheet/StyleSheetTypes.js

```diff
@@ -434,12 +434,15 @@ type ____LayoutStyle_Internal = $ReadOnly<{
    *  `width` and `height`. However, if there's not enough space,
    *  the component will shrink to its `minWidth` and `minHeight`.
    *
-   * flexGrow, flexShrink, and flexBasis work the same as in CSS.
+   * flexGrow, flexShrink, rowGap, columnGap, gap and flexBasis work the same as in CSS.
    */
   flex?: number,
   flexGrow?: number,
   flexShrink?: number,
   flexBasis?: number | string,
+  rowGap?: number,
+  columnGap?: number,
+  gap?: number,
```

Comment:
I reckon we want to put these in their own section rather than tied to `flex`. Maybe the description could be something like this:-

> In React Native, `gap` works the same way it does in CSS. If there are two or more children in a container, they will be separated from each other by the value of the gap - but the children will not be separated from the edges of their parent container.
> For horizontal gaps, use `columnGap`, for vertical gaps, use `rowGap`, and to apply both at the same time, it's `gap`.
> When `align-content` or `justify-content` are set to `space-between` or `space-around`, the gap specifies the minimum distance between children.
> See https://developer.mozilla.org/en-US/docs/Web/CSS/gap for more details.

Reply:
This looks better. @NickGerleman wdyt?

Reply:
This change makes sense to me, to separate from the above which describe how flexible lengths work 👍